sdcm/utils/docker.py

```diff
@@ -216,7 +226,14 @@ def destroy_container(cls, instance: object, name: str, ignore_keepalive: bool =
     @classmethod
     def destroy_all_containers(cls, instance: object, ignore_keepalive: bool = False) -> None:
         for name in tuple(instance._containers.keys()):
-            cls.destroy_container(instance, name, ignore_keepalive=ignore_keepalive)
+            try:
+                cls.destroy_container(instance, name, ignore_keepalive=ignore_keepalive)
+            except Exception:  # pylint: disable=broad-except
```

Comment:
please notice, that we don't track those event on other steps like clear-resources and such.

Reply:
Ok, I've made it little bit noisier.

Reply:
@fruch where will those events be saved? will they be added to `critical.log`?

Reply:
Looks like no where...